rgeo.gemspec

```diff
@@ -25,6 +25,7 @@ Gem::Specification.new do |spec|
   spec.files = Dir["lib/**/*.rb", "ext/**/*.{rb,c,h}", "LICENSE.txt", "README.md", ".yardopts"]
   spec.extensions = Dir["ext/*/extconf.rb"]
 
+  spec.add_development_dependency "base64"
```

Comment:
Why do we need to add this?

Reply:
required by Ruby 3.4.0 - test suite failed because of this also for jruby

Reply:
I would add it to the Gemfile as it is not needed for development per se. WDYT?